src/plotman/configuration.py

```diff
@@ -89,6 +89,7 @@ class Plotting:
     job_buffer: int
     farmer_pk: Optional[str] = None
     pool_pk: Optional[str] = None
+    x: Optional[bool] = None
```

Comment:
We don't actually need this to be optional I don't think, just a default value cuts it.

```suggestion
    x: bool = False
```